data/fixtures/scopes/textual/boundedParagraph2.scope

```diff
@@ -0,0 +1,211 @@
+aaa
+bbb {
+    ccc
+    ddd
+
+    eee
+    fff
+
+    ggg
+    hhh
+} iii
+jjj
+
+kkk
+---
+
+[#1 Content] =
+[#1 Domain] = 0:0-3:7
+  >---
+0| aaa
+1| bbb {
+2|     ccc
+3|     ddd
+   -------<
+
+[#1 Removal] = 0:0-5:0
+  >---
+0| aaa
+1| bbb {
+2|     ccc
+3|     ddd
+4|
+5|     eee
+   <
+
+[#1 Trailing delimiter: Content] = 4:0-4:0
+  ><
+4|
+[#1 Trailing delimiter: Removal] = 4:0-5:0
+  >
+4|
+5|     eee
+   <
+
+[#1 Insertion delimiter] = "\n\n"
+
+
+[#2 Content] =
+[#2 Domain] = 2:4-3:7
+      >---
+2|     ccc
+3|     ddd
+   -------<
```

Comment:
as above, I think domain here should extend to at least start of line, maybe all the way to end of opening `{`